src/config/locales.md

```diff
@@ -9,26 +9,16 @@ locale -a
 ## Enabling locales
 
 To enable the locale `xxxx`, un-comment or add the relevant line in
-`/etc/default/libc-locale.conf` and run
-
-```
-# xbps-reconfigure -f glibc-locales
-```
+`/etc/default/libc-locale.conf` and
+[reconfigure](../xbps/packages/management.md) the `glibc-locales` package.
 
 ## Setting the system locale
```

Comment:
This isn't so much related to your changes but looking at this, I think it could be reorganized to 'set' the locale first, and then mention that it must then be enabled.